client/webpack.config.js

```diff
@@ -56,7 +56,10 @@ module.exports = {
     new webpack.HotModuleReplacementPlugin(),
     new SystemBellPlugin(),
     new webpack.DefinePlugin({
-      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+      'process.env': {
+        'NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+        'LUMEN_KEYCLOAK_URL': JSON.stringify(process.env.LUMEN_KEYCLOAK_URL)
```

Comment:
@iperdomo The stings are inserted verbatim from the environment **at build time**. Without JSON.stringify we would end up with something like

```
url: http://localhost:8080
```

instead of

```
url: "http://localhost:8080"
```